### PR TITLE
L m n o tiggers

### PR DESCRIPTION
**Pull request by fatima99s**

### CODE REVIEW

File: plugins/aws/msk/mskClusterEncryptionAtRest.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ module.exports = {
             default: 'awscmk',
         }
     },
+    realtime_triggers: ['kafka:CreateCluster'],
```
Comment (AkhtarAmir) [inline plugins/aws/msk/mskClusterEncryptionAtRest.js:21]: KMS events?

File: plugins/aws/msk/mskClusterUnauthAccess.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ module.exports = {
     link: 'https://docs.aws.amazon.com/msk/latest/developerguide/msk-authentication.html',
     recommended_action: 'Ensure that MSK clusters does not have unauthenticated access enabled.',
     apis: ['Kafka:listClusters'],
+    realtime_triggers: ['kafka:CreateCluster','kafka:UpdateClusterConfiguration'],
```
Comment (AkhtarAmir) [inline plugins/aws/msk/mskClusterUnauthAccess.js:14]:
```suggestion
    realtime_triggers: ['kafka:CreateCluster','kafka:UpdateSecurity'],
```

File: plugins/aws/mwaa/environmentDataEncrypted.js [hunk 3]
```diff
@@ -19,6 +19,7 @@ module.exports = {
             default: 'awscmk'
         }
     },
+    realtime_triggers: ['mwaa:CreateEnvironment'],
```
Comment (AkhtarAmir) [inline plugins/aws/mwaa/environmentDataEncrypted.js:22]: KMS events?

File: plugins/aws/neptune/neptuneDBInstanceEncrypted.js [hunk 4]
```diff
@@ -20,6 +20,7 @@ module.exports = {
             default: 'awscmk',
         }
     },
+    realtime_triggers: ['neptune:CreateDBCluster'],
```
Comment (AkhtarAmir) [inline plugins/aws/neptune/neptuneDBInstanceEncrypted.js:23]: KMS events

Comment (AkhtarAmir) [reply plugins/aws/neptune/neptuneDBInstanceEncrypted.js:23]: Update event?

File: plugins/aws/openSearchServerless/opensearchCollectionCmkEncrypted.js [hunk 5]
```diff
@@ -19,6 +19,8 @@ module.exports = {
             default: 'awscmk'
         }
     },
+    realtime_triggers: ['opensearchserverless:CreateCollection'],
```
Comment (AkhtarAmir) [inline plugins/aws/openSearchServerless/opensearchCollectionCmkEncrypted.js:22]: KMS events?

File: plugins/aws/openSearchServerless/opensearchCollectionPublicAccess.js [hunk 6]
```diff
@@ -10,6 +10,7 @@ module.exports = {
     link: 'https://docs.aws.amazon.com/opensearch-service/latest/developerguide/serverless-network.html',
     recommended_action: 'Update the network policy and remove the public access to the collection.',
     apis: ['OpenSearchServerless:listNetworkSecurityPolicies', 'OpenSearchServerless:getNetworkSecurityPolicy', 'OpenSearchServerless:listCollections'],
+    realtime_triggers: ['opensearchserverless:CreateCollection', 'opensearchserverless:CreateSecurityPolicy', 'opensearchserverless:UpdateSecurityPolicy','opensearchserverless:DeleteSecurityPolicy'],
```
Comment (AkhtarAmir) [inline plugins/aws/openSearchServerless/opensearchCollectionPublicAccess.js:13]: Why CreateSecurityPolicy

**Review by AkhtarAmir — CHANGES_REQUESTED** [review]

**Review by alphadev4 — APPROVED** [review]

**Pull request merged**